nemo/collections/nlp/models/dialogue/dialogue_s2s_generation_model.py

```diff
@@ -273,8 +274,14 @@ def prepare_data(self):
             self.dialogues_processor = DialogueMellonQADataProcessor(
                 data_dir=self._cfg.dataset.data_dir, tokenizer=self.tokenizer, cfg=self._cfg.dataset
             )
+        elif self._cfg.dataset.task == "custom_response_generation":
```

Comment:
Please also add "custom_response_generation" to dialogue_config.yaml as one possible task